src/components/App/App.tsx

```diff
@@ -37,7 +16,7 @@ class App extends React.Component<
           {
             // TO DO: add router config for this route
           }
-          <Route path="/story/:id" component={MerchantPage} />
+          <Route path="/story/:id" component={SellerPage} />
```

Comment:
Is it possible to do :id/story? It looks funny in the URL this way, and if we're linking people here I don't want the URL to change

Reply:
Actually, can we change it to just :id ? The other tabs can have :id/menu and :id/photos